example/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="jquery.min.js"></script>
+    <script src="semantic.min.js"></script>
```

Comment:
I wasn't sure whether jquery.min.js was actually required or not.  If it is, then we should add it to js-sources as well and hopefully be able to remove it from here too.